packages/rabbitmq/src/rabbitmq.interfaces.ts

```diff
@@ -95,6 +95,7 @@ export interface MessageHandlerOptions {
   assertQueueErrorHandler?: AssertQueueErrorHandler;
   allowNonJsonMessages?: boolean;
   createQueueIfNotExists?: boolean;
+  usePersistentReplyTo?: boolean;
```

Comment:
I would add here some jsdoc with `@default false` because we should be transparent on those
If applicable, have a quick summary of this flag

Reply:
I've added the documentation